pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -261,6 +261,18 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 		return reconcile.Result{}, errors.Wrap(err, "failed to save cluster config")
 	}
 
+	if cephBlockPoolRadosNamespace.Spec.Mirroring.Enabled {
+		err = cephclient.EnableRBDRadosNamespaceMirroring(r.context, r.clusterInfo, cephBlockPoolRadosNamespace.Spec.BlockPoolName, getRadosNamespaceName(cephBlockPoolRadosNamespace), cephBlockPoolRadosNamespace.Spec.Mirroring.RemoteNamespace, cephBlockPoolRadosNamespace.Spec.Mirroring.Mode)
```

Comment:
Where do we enable mirroring of the default namespace?

Reply:
@travisn we dont support creating the default namespace in Rook isn't it? do we need to support any operation related to default namespace (which is empty string `""`) ?

Reply:
After discussion, I got to know,
If we enable pool mirroring the deafult namesapce mirroring is auto eanbled, So no need to have a separate support